SixTest/elensidealthck4d/README

```diff
@@ -1,3 +1,9 @@
+Tests:
+- thin4d and thin6d use the same single and structure elements. Test based on LHC lattice with HEL inserted at IP4.
+- thck4d adds an element 'thicky' with a nonzero length to force the thick 4D tracking to the thin4d/thin6d test
+- thck6d is based on thick6dblocks as the LHC example could not be used as for 6D thick tracking linear elements can not be grouped in blocks -> LHC example as defined fails.
+
+Pythong script:
```

Comment:
Look at the line I'm commenting on - "Pythong"?

HEL/ELEN: Yeah, I didn't spot that. Also punctuation, add a period at the end of `thck4d` like it is added for the other ones. Thanks!